packages/cli/src/rollup-plugins/flow-and-prod-dev-entry.ts

```diff
@@ -56,7 +56,10 @@ export default function flowAndNodeDevProdEntry(
         if (!resolved.external && !allowedExtensionRegex.test(resolved.id)) {
           warnings.push(
             new FatalError(
-              `only .ts, .tsx, .js, .jsx, and .json files can be imported but "${source}" is imported in ${
+              `only .ts, .tsx, .js, .jsx, and .json files can be imported but "${path.relative(
```

Comment:
both of those added `path.relative` calls could potentially be hoisted (the call is the same), maybe they even need to be wrapped with `normalizePath` but I'm not sure if that's required here yet. They might also contain a bug right now because they assume that `importer` is defined but according to the types (and some of the surrounding code here) it's actually nullable - the tests "pass" though and this doesn't crash. It would be good to add tests covering for the case when the `importer` is missing.